data/mur_citation_map.csv

```diff
@@ -0,0 +1,172 @@
+2:431,52:30101
```

Comment:
My eyes could be deceiving me, but I still see duplicates. Lines 1-4, for instance.